xwf/gateway/deploy/roles/xwfm/templates/ifcfg-uplink_br0

```diff
@@ -4,4 +4,4 @@ DEVICETYPE=ovs
 TYPE=OVSBridge
 BOOTPROTO=static
 HOTPLUG=no
-OVS_EXTRA="set-fail-mode $DEVICE secure"
+OVS_EXTRA="set-fail-mode $DEVICE secure -- set-controller $DEVICE tcp:{{ xwf_ctrl_ip }}:6653"
```

Comment:
dont we need similar change for cwag_br0?

Reply:
@pshelar  this is set in cwag when the pipelined docker comes up in the init script of the docker.